dsl_parser/elements/policies.py

```diff
@@ -285,6 +298,16 @@ class PolicyInstanceTargets(Element):
     required = True
     schema = List(type=PolicyInstanceTarget)
 
+    def parse(self, **kwargs):
+        """
+        This is a patch due to a problem in the infrastructure.
```

Comment:
Explain what the problem is and how does this patch fix it.

Reply:
Added